src/custom/utils/signatures.ts

```diff
@@ -137,16 +137,19 @@ async function _signPayload(
   payload: any,
   signFn: typeof _signOrder | typeof _signOrderCancellation,
   signer: Signer,
-  signingMethod: 'v4' | 'int_v4' | 'v3' | 'eth_sign' = 'v4'
+  signingMethod: 'default' | 'v4' | 'int_v4' | 'v3' | 'eth_sign' = 'v4'
```

Comment:
We now allow to specify the "default" version. In this case, will be same as v4, at least now. In the future who knows :) 

Setting to `default` is what makes us to send the RPC call with `eth_signTypedData` instead of `eth_signTypedData_v4`